src/widgets/splits/SplitHeader.cpp

```diff
@@ -366,6 +366,12 @@ std::unique_ptr<QMenu> SplitHeader::createMainMenu()
         menu->addAction(OPEN_PLAYER_IN_BROWSER, this->split_,
                         &Split::openBrowserPlayer);
 #endif
+        if (this->split_->getChannel()->hasModRights())
+        {
+            menu->addAction(OPEN_MOD_VIEW_IN_BROWSER, this->split_,
+                            &Split::openModViewInBrowser);
+        }
+
```

Comment:
nitpick: Why is this inserted in between "Open stream in X" options? Wouldn't it make more sense to move it under "Open in streamlink"?
Since this is not going to be present in all the splits, it might be confusing to my (and maybe other's) muscle memory while trying to open a new stream in streamlink with one click.

Reply:
@zneix Good point, and I definitely agree! I've now moved it to be below both player options.